articles/tutorials/get-user-information-with-unbounce-landing-pages.md

```diff
@@ -3,64 +3,67 @@
 ### Configuration on Auth0
 
 1. Create an account in [Auth0](https://auth0.com)
-2. Go to **Applications -> NEW**. Crete an HTML5 Application on Auth0 dashboard and go to Settings and take note of the Client ID and Domain.
+2. Go to **Clients -> + Create Client**. Crete a Single Page Application on the Auth0 dashboard, go to Settings and take note of the Client ID and Domain. Also, add the `callback URL` in both `Allowed Callback URLs` and `Allowed Origins (CORS)` (it should be your unbounce page URL. For example:`http://unbouncepages.com/changeit`).
 3. Go to **Connections -> Social** and Turn on the Social Providers you want to support.
 
-![](/media/articles/scenarios/unbounce/social-connections.png)
+> NOTE: you should configure the Client ID and Secret for each social connection.
 
-> NOTE: if you want to show your own logo on the consent page of the social provider, click on the provider logo and it will be expanded. Click on How to obtain a CLient ID link for instructions.
+![](/media/articles/scenarios/unbounce/social-connections.png)
 
 ## Configuration on Unbounce
 
-1. Add a new JavaScript to your Unbounce landing page and replace the Auth0 domain and client id with the values you've got from Auth0. Also make sure to check jQuery as a dependency.
+* Add a button (or whatever UI element you consider) that will trigger the login with the provider. Take note of the button ID under `Properties`->`Element Metadata`.
```

Comment:
@Annyv2 Maybe this `Properties`->`Element Metadata` should be using bold instead?